pylint/test/functional/inconsistent_returns.py

```diff
@@ -1,4 +1,4 @@
-#pylint: disable=missing-docstring, no-else-return, invalid-name, unused-variable, superfluous-parens
+#pylint: disable=missing-docstring, no-else-return, invalid-name, unused-variable, superfluous-parens, try-except-raise
```

Comment:
Curious that I'm not seeing any occurrence of the new error check in this file. Why was it disabled?

Reply:
That was probably when the new error check was also raised on non-bare raise. Thanks for catching this. I will update the code.